bundle_result.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package bundle_result;
+
+// Indicates the bundle successfully won its auction and was dispatched to the current leader.
+message Dispatched {
+  // The bundle's placement within the auction upon completion (e.g. 1st or 2nd place).
+  uint32 auction_placement = 1;
+
+  // This bundle's bid.
+  uint64 originator_bid_lamports = 2;
+
+  // Window of slots this bundle was sent for.
```

Comment:
what does this mean? should we just attach current slot?

Reply:
we forward bundles with a slight lookahead buffer [here](https://github.com/jito-labs/block-engine/blob/master/src/validator_interface_service/src/server.rs#L563)

Reply:
you think that's worth exposing? or should we just include curr slot number